plugins/rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -19,7 +19,7 @@ tags:
 - vulnerability
 hub_tags:
   use_cases: [asset_inventory, vulnerability_management]
-  keywords: [insightvm, rapid7, cloud]
+  keywords: [insightvm, rapid7, cloud, cloud_enabled]
```

Comment:
Can this be cloud enabled without support for connections?

Reply:
Unsure. I lack context for the cloud work on this plugin and will get feedback from Aine for why the `cloud_ready` tag is here.

Reply:
Removed along with `cloud_ready: true` tag.